.gitignore

```diff
@@ -166,3 +166,9 @@ Desktop.ini
 /slack/backend/app/build/
 /slack/backend/plugin/.gradle/
 /slack/backend/plugin/build/
+/amsterdam-email-api/backend/build/
```

Comment:
Can we add:
`**/.gradle` 
`**/build/`
`**/node_modules/`

instead?

Reply:
done